domain/admininfo.js

```diff
@@ -10,5 +10,10 @@ const admininfo = {};
 admininfo.getShirtOrders = function () {
```

Comment:
For me ```admininfo``` is more a service, then a domain object (as it doesn't contain any *business* logic but only the queries)